lib/prototype-rails/on_load_action_view.rb

```diff
@@ -14,9 +14,11 @@
   include ActionView::Helpers::ScriptaculousHelper
 end
 
-ActionView::TestCase.class_eval do
-  include ActionView::Helpers::PrototypeHelper
-  include ActionView::Helpers::ScriptaculousHelper
+if defined?(Rails) && Rails.env.test?
```

Comment:
Ah yes, that should work and is a bit safer, changed!

On Wed, Jan 21, 2015 at 7:48 AM, Rafael Mendonça França <
notifications@github.com> wrote:

> In lib/prototype-rails/on_load_action_view.rb
> <https://github.com/rails/prototype-rails/pull/33#discussion_r23308286>:
>
> > @@ -14,9 +14,11 @@
> >    include ActionView::Helpers::ScriptaculousHelper
> >  end
> >
> > -ActionView::TestCase.class_eval do
> > -  include ActionView::Helpers::PrototypeHelper
> > -  include ActionView::Helpers::ScriptaculousHelper
> > +if defined?(Rails) && Rails.env.test?
>
> I mean if defined?(Rails.env) && Rails.env.test?.
>
> I really thing we should not check any environment but I can't think in a
> better option besides a configuration.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rails/prototype-rails/pull/33/files#r23308286>.
>